src/integrations/admin/crawl-settings-integration.php

```diff
@@ -278,7 +278,7 @@ private function print_toggles( array $settings, Yoast_Form $yform, $is_network,
 				]
 			);
 			if ( $setting === 'remove_feed_global_comments_free' && ! $is_network ) {
-				echo '<p class="yoast-global-comments-feed-help-free">';
+				echo '<p class="yoast-crawl-settings-help">';
```

Comment:
There's a reference of the `yoast-global-comments-feed-help-free` class in admin-global.css still, but it looks like this IF is never being accessed (probably by mistake), so this change doesnt have any effect, but maybe best to clean the CSS rule anyway